src/app/(proper_react)/(redesign)/MobileShell.tsx

```diff
@@ -126,11 +126,7 @@ export const MobileShell = (props: Props) => {
                 <PageLink
                   href="/user/dashboard"
                   activeClassName={styles.isActive}
-                  onClick={() => {
-                    recordTelemetry("ctaButton", "click", {
-                      button_id: "navigation_dashboard",
-                    });
-                  }}
```

Comment:
It makes sense that this one would have the event name `link`, but in the [telemetry specs](https://www.figma.com/board/LGjPMlfA7Li3xkS5YZ6VLY/Monitor-Telemetry-Documentation?node-id=18-4160&t=7GuMAjXN2hXEjwFs-0) I see this one as `ctaButton`. With the new telemetry events this would just be a matter of making sure that we update the existing Figma specs accordingly (change the event name there from `ctaButton` to `link`).

In both cases, we would need to make sure to sync with PM before the telemetry changes and make sure that we don’t interfere with running telemetry.

Reply:
Request to PM for updated specs has been made.